lib/BackgroundJobs/IndexerJob.php

```diff
@@ -64,7 +64,7 @@ public function __construct(
 		private ITimeFactory $timeFactory,
 	) {
 		parent::__construct($time);
-		$this->setInterval($this->getMaxIndexingTime());
+		$this->setInterval(5 * 60);
```

Comment:
Should we change this above in the class constant instead to be consistent, or did you purposely only change it here?

Reply:
yeah it's only done here on purpose. If there is only one cron runner, it spawns jobs every 5 mins and max running jobs at any time is ~10. The max indexing time is to stop a particular executing job from continuing further than 30 mins, how I am looking at the values.
We can make it a little less aggressive with 10 mins interval and also make it configurable.